src/group_theory/commutator.lean

```diff
@@ -21,7 +21,16 @@ is the subgroup of `G` generated by the commutators `h₁ * h₂ * h₁⁻¹ * h
 * `⁅H₁, H₂⁆` : the commutator of the subgroups `H₁` and `H₂`.
 -/
 
-variables {G G' F : Type*} [group G] [group G'] [monoid_hom_class F G G] (f : F) (g₁ g₂ g₃ : G)
+variables {G G' F : Type*} [group G] [group G'] [monoid_hom_class F G G] (f : F) (g g₁ g₂ g₃ : G)
+
+lemma commutator_element_eq_one_iff : ⁅g₁, g₂⁆ = 1 ↔ g₁ * g₂ = g₂ * g₁ :=
```

Comment:
Does it make sense to phrase this in terms of `commute g₁ g₂`? There is a nice API for `commute`.

Reply:
Is it ok if I have both versions?